karma.conf.js

```diff
@@ -5,24 +5,9 @@ var absoluteBasePath = path.resolve(__dirname);
 /* global process */
 
 // configures browsers to run test against
-// any of [ 'ChromeHeadless', 'Chrome', 'Firefox', 'IE', 'PhantomJS' ]
+// any of [ 'ChromeHeadless', 'Chrome', 'Firefox', 'IE']
 var browsers =
-  (process.env.TEST_BROWSERS || 'PhantomJS')
-    .replace(/^\s+|\s+$/, '')
-    .split(/\s*,\s*/g)
-    .map(function(browser) {
-      if (browser === 'ChromeHeadless') {
-        process.env.CHROME_BIN = require('puppeteer').executablePath();
```

Comment:
What's the consequence of completely dropping puppeteer? In most projects we have it in there, still (cf. https://github.com/bpmn-io/bpmn-js/blob/develop/test/config/karma.unit.js#L11 for example).

I see however that we do not have it in dmn-js (https://github.com/bpmn-io/dmn-js/blob/develop/packages/dmn-js-shared/karma.base.js#L9).

Reply:
Using puppeteer is also the preferred way to run it against CI: https://github.com/karma-runner/karma-chrome-launcher#headless-chromium-with-puppeteer.

Reply:
Good question. IMO:

1. Practically I cannot see impact (neither on travis nor locally) (same goes for dmn-js library)
2. However, puppeteer brings its own ChromeHeadless version. So using puppeteer would bring more stability I suppose.

Ultimately: I feel we should be consistent here (as you indicated). Therefore:
* I would let this repo (which is cloesly tied to dmn-js) follow dmn-js. Therefore drop puppeteer
* I would create an issue to check across all our repos, how we use puppeteer, if we should use it, and then clean it up (with a general guideline)

WDYT?

Reply:
Sounds good.

Reply:
If we do not see an impact we should indeed converge to "no puppeteer" with all our projects.

Reply:
It has the additional benefit of significantly decreased local install times.

It will fail any build if no local chrome is installed, though.

Reply:
> It will fail any build if no local chrome is installed, though.

That is the point I think. 

I will create an issue and we can decide and harmonize this for all repos

Reply:
https://github.com/bpmn-io/internal-docs/issues/240